analyzere/base_resources.py

```diff
@@ -48,6 +53,43 @@ def __deepcopy__(self, memo):
             return copy.deepcopy(self.__wrapped__, memo)
         return Reference(self._href)
 
+    def __getattribute__(self, name):
+        # To see if name is an instanced attribute of Reference. Here we can
+        # check the names of the class attributes and then return the instance
+        # attribute with the same name.
+
+        attr = Proxy.__getattribute__(self, name)
+        if hasattr(Reference, name):
+            return attr
```

Comment:
Doesn't this skip anything that is a class member of `Reference` (and either not a member of `Proxy`, or overridden)?

For example, if you get `Reference.__copy__()`, I think this will return `Proxy`'s `__copy__()`.